.editorconfig

```diff
@@ -4,7 +4,5 @@ root = true
 ; Unix-style newlines
 [*]
 end_of_line = LF
-
-[*.php]
```

Comment:
It's seem this file haven't been serving its purpose before.
This allows to have standard 4-space format for each file.